assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -26,92 +26,36 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import {
+  Banner,
+  Checkbox,
+  Text,
+  THEME_CONSTANTS,
+} from '../../../../design-system';
 import { TranslateWithMarkup } from '../../../../i18n';
-import { TypographyPresets, useLayoutContext } from '../../../components';
-import { Close as CloseSVG } from '../../../icons';
+import { useLayoutContext } from '../../../components';
 import { useConfig } from '../../config';
+import { APP_ROUTES } from '../../../constants';
 import useTelemetryOptIn from './useTelemetryOptIn';
 
-const Banner = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  margin: 0 20px;
-  padding-top: 24px;
-  background-image: url('${({ backgroundUrl }) => backgroundUrl}');
-  background-size: cover;
-  border-radius: 8px;
-`;
-
-const Header = styled.div`
-  width: 100%;
-  text-align: center;
-`;
-
-const Title = styled.h1`
-  display: inline-block;
-  font-size: 18px;
-  font-weight: ${({ theme }) =>
-    theme.DEPRECATED_THEME.typography.weight.normal};
-  line-height: 24px;
-  margin-bottom: 13px;
-`;
-
 const Label = styled.label.attrs({ htmlFor: 'telemetry-banner-opt-in' })`
   display: flex;
 `;
 
-export const LabelText = styled.span`
-  ${TypographyPresets.Small};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray400};
+const LabelText = styled(Text)`
   margin-bottom: 16px;
-  max-width: 600px;
+  margin-left: 8px;
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  letter-spacing: 0.0133em;
```

Comment:
I've been noticing some rogue letter spacings in the designs too. Is this something we should add tokens for in the design system @BrittanyIRL ?

Reply:
yeah i've been wondering when these were added to the designs since the letter spacing should really be consistent within the text presets defined in the theme. Made a new ticket: https://github.com/google/web-stories-wp/issues/6174